script/Arcueid/Day 10/10_00_ARC10_1.txt

```diff
@@ -787,7 +787,7 @@ Arcueid bounds over to the window that I'd suspected she broke in through.
 -- Page 56, Offset 24403.
 -- 「あー、呼んだけど呼んでない！　止めて悪かった、外に出ていてくれ。すぐに追いつくから」
 "Yeah, I did, but I didn't!
- Sorry for jerking you around, but you can head on outside.
+ Sorry for interrupting you, but you can head on outside.
```

Comment:
This loses some meaning - he's jerking her around (leave, actually hold on) not just interrupting her action.